specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-09-01/dataConnectorDefinitions.json

```diff
@@ -0,0 +1,706 @@
+          "type": "string"
+        },
+        "templateSpecVersion": {
+          "description": "Gets or sets the template version.",
+          "type": "string"
+        }
+      }
+    },
+    "CustomizableConnectorUiConfig": {
+      "description": "The UiConfig for 'Customizable' connector definition kind.",
+      "required": [
+        "dataTypes",
+        "descriptionMarkdown",
+        "graphQueries",
+        "instructionSteps",
+        "permissions",
+        "publisher",
+        "title",
+        "connectivityCriteria"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Gets or sets custom connector id. optional field.",
+          "type": "string"
+        },
+        "title": {
+          "description": "Gets or sets the connector blade title.",
+          "type": "string"
+        },
+        "publisher": {
+          "description": "Gets or sets the connector publisher name.",
+          "type": "string"
+        },
+        "descriptionMarkdown": {
+          "description": "Gets or sets the connector description in markdown format.",
+          "type": "string"
+        },
+        "graphQueries": {
+          "description": "Gets or sets the graph queries to show the current data volume over time.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GraphQuery"
+          },
+          "x-ms-identifiers": []
+        },
+        "dataTypes": {
+          "description": "Gets or sets the data types to check for last data received.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConnectorDataType"
+          },
+          "x-ms-identifiers": []
+        },
+        "connectivityCriteria": {
+          "description": "Gets or sets the way the connector checks whether the connector is connected.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConnectivityCriterion"
+          },
+          "x-ms-identifiers": []
+        },
+        "availability": {
+          "$ref": "#/definitions/ConnectorDefinitionsAvailability"
+        },
+        "permissions": {
+          "$ref": "#/definitions/ConnectorDefinitionsPermissions"
+        },
+        "instructionSteps": {
+          "description": "Gets or sets the instruction steps to enable the connector.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstructionStep"
+          },
+          "x-ms-identifiers": []
+        },
+        "logo": {
+          "description": "Gets or sets the connector logo to be used when displaying the connector within Azure Sentinel's connector's gallery.\r\nThe logo value should be in SVG format.",
+          "type": "string"
+        },
+        "isConnectivityCriteriasMatchSome": {
```

Comment:
>isConnectivityCriteriasMatchSome [](http://example.com/codeflow?start=9&length=32)

Recommend enum instead of boolean, e.g. criteriaCondition: { And, Or }